backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -473,9 +457,6 @@ def forward(self, x, y):
                 return self.sub(x, y)
 
         instance = Sub()
-        example_inputs = (torch.randn(2, 5, 1, 3), torch.randn(2, 5, 1, 3))
-        self.lower_module_and_test_output(instance, example_inputs)
```

Comment:
why is this removed?

Reply:
Both test cases were work, remove it here because we hope each test case produces only one artifact, for the ease of our local testing.
We will make the test as comprehensive as possible when we start doing CI integration, thank you.